code/defines/obj/weapon.dm

```diff
@@ -860,7 +860,7 @@
 
 /obj/item/weapon/ectoplasm
 	name = "ectoplasm"
-	desc = "The remnants of a being between the world of the dead and that of the living. Spooky."
```

Comment:
That was a change I made. It was previously just `desc = "spooky"`. I suppose this could be a useful description.